<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Помощник в принятии решений</string>
  </property>
  <widget class="QPushButton" name="button_positive">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>За</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_negative">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Против</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_argument">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Аргумент</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scroll_positive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>370</width>
     <height>370</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>368</width>
      <height>368</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scroll_negative">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>370</width>
     <height>370</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>368</width>
      <height>368</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_positive">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Голосов:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_negative">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>180</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Голосов:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_question">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вопрос</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_reset">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Сброс</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_question">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>300</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="placeholderText">
    <string>Вопрос</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ask">
   <property name="geometry">
    <rect>
     <x>355</x>
     <y>20</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Задать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
